### ISSUE TITLE
cmd/gofmt: indenting of multi-line argument list is confusing

### ISSUE THREAD

**Issue by pieter-biomerieux** [golang/go#48064]
### What version of Go are you using (`go version`)?

<pre>
$ go version
go version go1.17 linux/amd64
</pre>

### What did you do?

I pass this file to gofmt:

```go
func f(first int,
second string) {
fmt.Println(first, second, third)
fmt.Println("That's all folks")
}
```

### What did you see?
```go
func f(first int,
	second string) {
	fmt.Println(first, second, third)
	fmt.Println("That's all folks")
}

```
Notice how the argument list blends into the function's code.

### What did you expect?
I can think of two options:
- Add two indentation levels when starting the second line of the multi-line argument list, and remove one indentation level when entering the function
```go
func f(first int,
		second string) {
	fmt.Println(first, second, third)
	fmt.Println("That's all folks")
}
```
- Align with the first argument of the argument list
```go
func f(first int,
       second string) {
    fmt.Println(first, second, third)
    fmt.Println("That's all folks")
}
```

**Comment by pieter-biomerieux** [comment]
I don't know the logic of gofmt. This might be the same issue as #48065. Sorry to double submit if this is the case.

**Comment by cherrymui** [comment]
I think we always indent a tab if the second line is the same statement following the first line. So this is working as intended.

cc @griesemer  to confirm. Thanks.

**Comment by griesemer** [comment]
What @cherrymui said.

**Comment by pieter-biomerieux** [comment]
I would think the goal of gofmt is to produce easily readable code. As shown by my example  "always indent a tab if the second line is the same statement following the first line" is clearly not a great rule.

I didn't want to submit this as a bug. I searched for ways to submit improvements, but couldn't find anything. So I submitted it here as an issue. In most other projects, github issues are also used to discuss new feature/improvements. But seeing how quickly this get closed, I must have been wrong.

So how do I submit a request for an improvement to gofmt?
